ekncontent/ekncontent/eknc-domain.c

```diff
@@ -370,6 +370,38 @@ eknc_domain_process_subscription (EkncDomain *self,
 }
 
 static gboolean
+eknc_domain_import_subscriptions (EkncDomain *self,
+                                  GFile *subscriptions_dir,
+                                  JsonArray *json_subscriptions,
+                                  const gchar *relative_path,
+                                  GCancellable *cancellable,
+                                  GError **error)
+{
+  g_autoptr(GFileEnumerator) subscriptions_iter = g_file_enumerate_children (subscriptions_dir, "",
+                                                                             G_FILE_QUERY_INFO_NONE,
+                                                                             cancellable, error);
+  while (subscriptions_iter != NULL)
+    {
+      GFile *dir = NULL;
+      GFileInfo *info = NULL;
+      g_file_enumerator_iterate (subscriptions_iter, &info, &dir, cancellable, error);
+      if (dir == NULL)
```

Comment:
nit: if g_file_enumerator_iterate() returns false then there was an error
```
if(!g_file_enumerator_iterate (subscriptions_iter, &info, &dir, cancellable, error))
    break;
```

Reply:
good catch, I think it should `return FALSE;` in that case, to propagate the error properly.